website/docs/r/kusto_cluster.html.markdown

```diff
@@ -46,14 +46,20 @@ The following arguments are supported:
 
 * `sku` - (Required) A `sku` block as defined below.
 
+* `allowed_ips` - (Optional) The list of ips in the format of CIDR allowed to connect to the cluster.
+
 * `double_encryption_enabled` - (Optional) Is the cluster's double encryption enabled? Defaults to `false`. Changing this forces a new resource to be created.
 
+* `fqdns` - (Optional) List of allowed FQDNs(Fully Qualified Domain Name) for egress from Cluster.
```

Comment:
how do we distinguish the cases of empty list = nothing is allowed, vs everything is allowed?

Reply:
as in `fqdns = []` = nothing, and not setting `fqdns`s at all?